docs/office-hours.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 7
+title: ☎️ Office Hours
+---
+
+**Ecosystem Funding Office Hours** are a chance to ask the W3F Ecosystem Funding team questions regarding a specific project or (potential) grant application. It offers
+
+- general guidance regarding the various grants programs and prizes,
+- some quick initial feedback and
+- help how to navigate the ecosystem.
+
+Apply for Office Hours if you
+
+- want to find out what kind of support there might be available for your needs,
+- need feedback before submitting an application or
+- look for help finding other resources you might need.
+
+It is **not** a chance to _pitch_ your project, especially since only the various programs and prizes have different evaluation criteria and decision makers and only a small subset of the Grants team will participate in the call. To apply, please fill out the [Office Hours :alarm_clock: form](https://forms.gle/54xkiqU37WwdN9UR6). Be as specific as possible, so we can help you more quickly. We will get back to you with follow-up questions or a link for booking a timeslot.
```

Comment:
```suggestion
It is **not** a chance to _pitch_ your project, especially since only the various programs and prizes have different evaluation criteria and decision makers and only a small subset of the W3F Ecosystem Funding team will participate in the call. To apply, please fill out the [Office Hours :alarm_clock: form](https://forms.gle/54xkiqU37WwdN9UR6). Be as specific as possible, so we can help you more quickly. We will get back to you with follow-up questions or a link for booking a timeslot.
```